lib/Mojo/Cookie/Response.pm

```diff
@@ -56,6 +55,9 @@ sub to_string {
   # "Max-Age"
   if (defined(my $max = $self->max_age)) { $cookie .= "; Max-Age=$max" }
 
+  # "SameSite"
+  if (defined(my $samesite = $self->samesite)) { $cookie .= '; SameSite'.ucfirst($samesite) }
```

Comment:
I believe this should be:

`if (defined(my $samesite = $self->samesite)) { $cookie .= '; SameSite='.ucfirst($samesite) }`

Reply:
Although, if you're fixing up the casing, you may want `ucfirst(lc($samesite))`.